charts/rstudio-connect/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 name: rstudio-connect
 description: Official Helm chart for RStudio Connect
-version: 0.2.39-alpha03
+version: 0.2.39-alpha04
 apiVersion: v2
-appVersion: 2022.07.0-dev-428
```

Comment:
similarly, why is there a `-dev-428` suffix on here? is that normal, or would we usually have something like `2022.06.2` since that was the last released version?

Reply:
Same explanation as above 😄 Yes - normally you would have seen 2022.06.2 getting removed